org.knime.knip.base/src/org/knime/knip/base/nodes/util/tilelooper/TileIteratorLoopEndNodeFactory.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE knimeNode PUBLIC "-//UNIKN//DTD KNIME Node 2.0//EN" "http://www.knime.org/Node.dtd">
+<knimeNode icon="tileiteratorend.png" type="LoopEnd">
+	<name>Tile Loop End</name>
+
+	<shortDescription>
+		Collects tiles of Tile Loop Start and recombines them to images.
+	</shortDescription>
+
+	<fullDescription>
+		<intro>
+			Merges the tiles of a Tile Loop Start into the original images.
```

Comment:
Should be the same as the short description, I think by merging these two sentences together we'll get the best result.